ytmusicapi/auth/oauth/base.py

```diff
@@ -0,0 +1,96 @@
+from typing import Optional, Any, Union, Dict
```

Comment:
return value type hints are missing in this file (except for `Credentials` class)